### PR TITLE
perlstyle.pod: Fix wording

### PR DESCRIPTION
**Pull request by rwp0**
Spelling-only.

### CODE REVIEW

File: pod/perlstyle.pod
Comment (khwilliamson) [inline pod/perlstyle.pod]: 'your' was correct

**Comment by rwp0** [comment]
Please squash before merging 

GitHub UI should allow this with the merge button

File: pod/perlstyle.pod
Comment (khwilliamson) [inline pod/perlstyle.pod]: including 'about' is more correct than removing it, as its use indicates there may be other things not listed here, which gives wiggle room to us if something comes up

**Review by khwilliamson — CHANGES_REQUESTED** [review]

File: pod/perlstyle.pod
Comment (rwp0) [reply pod/perlstyle.pod]: But the double "about" sounds weird

File: pod/perlstyle.pod
Comment (rwp0) [reply pod/perlstyle.pod]: Yes, see commit 2

It was a technical error from my side.

File: pod/perlstyle.pod
Comment (rwp0) [reply pod/perlstyle.pod]: @khwilliamson ok I added the "about" back, now it's only "lay out" that's wrong.

Could you squash and merge via GitHub or should I squash the three commits myself?

Comment (khwilliamson) [reply pod/perlstyle.pod]: I can do the squash, but it's a pain for me.  It's should be easier for you.  If you don't know how to do it, it would be good to learn for future use.  Someone can guide you through.  email me to get started.

Comment (rwp0) [reply pod/perlstyle.pod]: Done (using GitHub Desktop). I can do via CLI too, but it's just easier.

Sorry, why it's a pain for you, because of GitHub UI?

There's no such a button in GitHub?:

https://docs.gitlab.com/ee/user/project/merge_requests/squash_and_merge.html

Thanks

---

This looks like it:

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits

**Comment by rwp0** [comment]
Since now it's just one character change, can we merge this already?

**Pull request merged**
